ui/src/main/java/com/doordeck/sdk/common/manager/Doordeck.kt

```diff
@@ -24,6 +23,7 @@ import com.doordeck.sdk.ui.qrcode.QRcodeActivity
 import com.doordeck.sdk.ui.unlock.UnlockActivity
 import com.doordeck.sdk.ui.unlock.UnlockActivity.Companion.COMING_FROM_DIRECT_UNLOCK
 import com.doordeck.sdk.ui.verify.VerifyDeviceActivity
+import com.github.doordeck.ui.BuildConfig
```

Comment:
Import needed?

Reply:
Yup. Namespaces were needed now and the `R` and `BuildConfig` classes changed
<img width="597" alt="image" src="https://github.com/doordeck/doordeck-sdk-java/assets/6842786/6bfb90ed-017d-47a8-be77-f36da47a6425">